charts/stable/valheim/questions.yaml

```diff
@@ -16,92 +16,74 @@ questions:
       type: dict
       attrs:
         - variable: SUPERVISOR_HTTP_USER
-          label: SUPERVISOR_HTTP_USER
+          label: Supervisor User
           schema:
             type: string
-            default: admin
+            default: ''
             required: true
         - variable: SUPERVISOR_HTTP_PASS
-          label: SUPERVISOR_HTTP_PASS
+          label: Supervisor PASS
           schema:
             type: string
             required: true
             private: true
-            default: REPLACETHIS
+            default: ''
         - variable: SERVER_PASS
-          label: SERVER_PASS
+          label: Server Pass
           schema:
             type: string
             required: true
             private: true
-            default: REPLACETHIS
+            default: ''
   - variable: env
     group: App Configuration
     label: Image Environment
     schema:
       additional_attrs: true
       type: dict
       attrs:
-        - variable: STATUS_HTTP
-          label: STATUS_HTTP
-          description: Turn on the status http server. Only useful on public servers (SERVER_PUBLIC=true).
-          schema:
-            type: boolean
-            default: true
-        - variable: SUPERVISOR_HTTP
-          label: SUPERVISOR_HTTP
-          description: Turn on supervisor's http server.
-          schema:
-            type: boolean
-            default: true
         - variable: SERVER_NAME
-          label: SERVER_NAME
+          label: Server Name
           description: Name that will be shown in the server browser.
           schema:
             type: string
             default: My Server
             required: true
         - variable: WORLD_NAME
-          label: WORLD_NAME
+          label: World Name
           description: Name of the world without .db/.fwl file extension.
           schema:
             type: string
             default: Dedicated
             required: true
         - variable: SERVER_PUBLIC
-          label: SERVER_PUBLIC
-          description: Whether the server should be listed in the server browser (true) or not (false).
+          label: Public Server
+          description: Whether the server should be listed in the server browser.
           schema:
             type: boolean
             default: true
         - variable: UPDATE_INTERVAL
-          label: UPDATE_INTERVAL
+          label: Update Interval
           description: How often we check Steam for an updated server version in seconds. This also controls how often Github is checked for ValheimPlus updates. Note that Github has an API rate limit of 60 req/hour. So setting this to less than 60 will result in errors if V+ is enabled.
           schema:
             type: int
             default: 10800
             required: true
         - variable: BACKUPS
-          label: BACKUPS
+          label: Backups
```

Comment:
These all seem like good UI improvements.  I don't know what they should have been.  But this one seems particularly bad to leave a single word.  `Enable Backups` or something?  I'm not sure which is why I left them the variable names...